.github/workflows/bundlesize.yml

```diff
@@ -0,0 +1,69 @@
+name: "Generate bundle-size for react charting"
+on:
+  schedule:
+    - cron: "30 3 * * *"
+  # "30 3 * * *" --> Every day at 3:30 AM UTC --> Every day at  9:00 AM IST    
+  workflow_dispatch:
+    inputs:
+      repo:
+        description: "Repo to run the tests on"
```

Comment:
Update descripton

Reply:
This is intended for manual runs.

Reply:
the description mentions run the tests. Update it to "Repo to run monosize on"

Reply:
Ok